indsl/filter/hilbert_huang_transform.py

```diff
@@ -0,0 +1,718 @@
+import logging
+
+from typing import Optional
+
+import numpy as np
+import pandas as pd
+import scipy.sparse as sp
+
+from PyEMD import EMD
```

Comment:
this should be inside the function https://indsl.docs.cognite.com/contribute.html#core-or-extras

Reply:
Is this completely copy paste code?